Source/WebCore/rendering/RenderListMarker.cpp

```diff
@@ -508,8 +508,8 @@ std::pair<int, int> RenderListMarker::layoutBoundForTextContent(String text) con
     if (style.lineHeight().isNormal()) {
         auto maxAscentAndDescent = ascentAndDescent(metricsOfPrimaryFont);
 
-        for (auto& fallbackFont : Layout::TextUtil::fallbackFontsForText(text, style, Layout::TextUtil::IncludeHyphen::No)) {
-            auto& fontMetrics = fallbackFont.fontMetrics();
+        for (WeakRef fallbackFont : Layout::TextUtil::fallbackFontsForText(text, style, Layout::TextUtil::IncludeHyphen::No)) {
```

Comment:
It seems like we should be using `Ref` instead?

Reply:
Sure.